scripts/linters/python_linter.py

```diff
@@ -417,23 +377,29 @@ def _lint_py_files(self, config_pylint, config_pycodestyle):
                 python_utils.PRINT('Linting Python files %s to %s...' % (
                     current_batch_start_index + 1, current_batch_end_index))
 
-            with linter_utils.redirect_stdout(stdout):
+            with linter_utils.redirect_stdout(sys.stdout):
```

Comment:
Why is this changed? Could you please let me know when is stdout used and when is sys.stdout used?

Reply:
Here stdout=python_utils.string_io(), we use _stdout_ when we need to both read and write to the stdout, and use _sys.stdout_ when we need to just write to the stdout. Here we do not need to read from the stdout so I removed the stdout.